Marlin/src/pins/stm32f1/pins_CREALITY_V452.h

```diff
@@ -0,0 +1,116 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * Creality v4.5.2 (STM32F103) board pin assignments
+ */
+
+#ifndef __STM32F1__
+  #error "Oops! Select an STM32F1 board in 'Tools > Board.'"
+#endif
+
+#if HOTENDS > 1 || E_STEPPERS > 1
+  #error "CREALITY supports up to 1 hotends / E-steppers. Comment out this line to continue."
+#endif
+
+#define BOARD_NAME "Creality v4.5.2"
+#define DEFAULT_MACHINE_NAME "Creality3D"
+
+//
+// EEPROM
+//
+#if NO_EEPROM_SELECTED
+  #define IIC_BL24CXX_EEPROM                      // EEPROM on I2C-0 used only for display settings
+  #if ENABLED(IIC_BL24CXX_EEPROM)
+    #define IIC_EEPROM_SDA                  PA11
+    #define IIC_EEPROM_SCL                  PA12
+    #define MARLIN_EEPROM_SIZE             0x800  // 2Kb (24C16)
+  #else
+    #define SDCARD_EEPROM_EMULATION               // SD EEPROM until all EEPROM is BL24CXX
+    #define MARLIN_EEPROM_SIZE             0x800  // 2Kb
+  #endif
+#endif
+
+
+//
+// Limit Switches
+//
+#define X_MIN_PIN           PC4
+// #define X_MAX_PIN          PA7
+#define Y_MIN_PIN           PC5
+#define Z_MIN_PIN           PA4
+#define COM_PIN             PA5
+
+//
+// Steppers
+//
+#define X_ENABLE_PIN        PC3
+#define X_STEP_PIN          PB8
+#define X_DIR_PIN           PB7
+
+#define Y_ENABLE_PIN        PC3
+#define Y_STEP_PIN          PB6
+#define Y_DIR_PIN           PB5
+
+#define Z_ENABLE_PIN        PC3
+#define Z_STEP_PIN          PB4
+#define Z_DIR_PIN           PB3
+
+#define E0_ENABLE_PIN       PC3
+#define E0_STEP_PIN         PC2
+#define E0_DIR_PIN          PB9
+
+
+//
+// Release PB4 (Z_STEP_PIN) from JTAG NRST role
+//
+#define DISABLE_DEBUG
+
+//
+// Temperature Sensors
+//
+#define TEMP_0_PIN         PB1   // TH1
+#define TEMP_BED_PIN       PB0   // TB1
+
+//
+// Heaters / Fans
+
+#define HEATER_0_PIN       PA1   // HEATER1
+#define HEATER_BED_PIN     PA2   // HOT BED
+
+#define FAN_PIN            PA0   // FAN
+#define FAN_SOFT_PWM
+
+/* SD card detect */
+#define SD_DETECT_PIN      PC7
+#define NO_SD_HOST_DRIVE  // This board's SD is only seen by the printer
+
+#define SDIO_SUPPORT      // Extra added by Creality
+#define SDIO_CLOCK 6000000 // In original source code overridden by Creality in sdio.h
+
+#define CASE_LIGHT_PIN PA6
+
+#define FIL_RUNOUT_PIN PA7
+#define PROBE_ENABLE_PIN    PC6 // Optoswitch to Enable Z Probe
+
+
+#define TEMP_TIMER_CHAN 4 // Channel of the timer to use for compare and interrupts
```

Comment:
I doubt this is actually needed.

Reply:
Referring to `TEMP_TIMER_CHAN`

Reply:
Maybe, I would need to follow it back a bit but for the moment all known references have it so I didnt touch it till I had time to investigate.

Reply:
It looks like it is already in the PIO environment, so I removed it here.
It probably isn't _actually_ needed anywhere, but removing it here should not matter.